physics/hubble_parameter.py

```diff
@@ -0,0 +1,75 @@
+"""
+Title : Calculating the Hubble Parameter
+
+Description : The Hubble parameter H is the Universe expansion rate in any time.
+In cosmology is costumary to use the redshift z in place of time, because
+the redshift is directily mensure in the light of galaxies moving away
+from us.
+
+So, the general relation that we obtain is
+
+H = H_0*(Omg_r*(z+1)**4 + Omg_m*(z+1)**3 + Omg_k*(z+1)**2 + Omg_l)**(1/2)
+
+where Omg_r,l,m are the relativity (the percentage) energy densities that exist
+in the Universe today. Here, Omg_m is the sum of the barion density and the
+dark matter. Omg_k is the curvature parameter and can be written in termm
+of the densities by the completness
+
+Omg_k = 1 - (Omg_m + Omg_r + Omg_l)
+
+Source :
+https://www.sciencedirect.com/topics/mathematics/hubble-parameter
+"""
+
+
+def hubble_parameter(H_0: float, Omg_r: float, Omg_m: float, Omg_l: float, z: float):
```

Comment:
Variable names should be words that humans can understand without looking at the comments.
```suggestion
def hubble_parameter(
    hubble_constant: float,
    radiation_density: float,
    mass_density: float,
    dark_energy_density: float,
    redshift: float,
) -> float:
```

Reply:
Do this